graders/python/python_autograder/code_feedback.py

```diff
@@ -467,12 +488,24 @@ def bad(msg: str) -> Literal[False]:
         return True
 
     @classmethod
-    def call_user(cls, f: Callable, *args: Any, **kwargs: Any) -> Any:
+    def call_user(
+        cls,
+        f: Callable,
+        stop_on_exception: bool = False,  # noqa: FBT001
```

Comment:
There's actually a pretty significant issue with the way this is set up right now. The argument after `f` given by client code is treated as `stop_on_exception`, and not one of the arguments in `*args`, so this is a major backwards incompatibility (I ran into this problem when testing locally). To resolve this, there are a couple of options:

- We could read `stop_on_exception` out of the `kwargs` dict and pass the remaining kwargs to the student function.
- We could add a separate wrapper function `call_user_stop_on_exception` that encodes the behavior we want.

Reply:
This may have been a bad merge. Can you move it after *args?

Reply:
It looks like that was there before? But I don't think moving it after works.

Reply:
It was reordered in [0bb1b24](https://github.com/PrairieLearn/PrairieLearn/pull/10590/commits/0bb1b246ed9e8a1a96b8fd4f8b40f231ae46a6be), and from https://github.com/PrairieLearn/PrairieLearn/pull/10590#issuecomment-2616792744 appears to work.